components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountDisableHandler.java

```diff
@@ -296,6 +308,16 @@ protected boolean handlePostSetUserClaimValues(Event event, String userName, Use
                     newAccountState = buildAccountState(AccountConstants.EMAIL_TEMPLATE_TYPE_ACC_DISABLED, userStoreManager,
                             tenantDomain, userName);
                 }
+                publishPostAccountDisabledEvent(IdentityEventConstants.Event.POST_DISABLE_ACCOUNT,
+                        event.getEventProperties(), true);
+                auditAccountDisable(AuditConstants.ACCOUNT_DISABLED, userName, userStoreDomainName,
+                        null, AuditConstants.AUDIT_SUCCESS,true);
+            } else if (disabledStates.DISABLED_UNMODIFIED.toString().equals(disabledState.get())) {
+                auditAccountDisable(AuditConstants.ACCOUNT_DISABLED, userName, userStoreDomainName,
+                        null, AuditConstants.AUDIT_SUCCESS,false);
+            } else if (disabledStates.ENABLED_UNMODIFIED.toString().equals(disabledState.get())) {
+                auditAccountDisable(AuditConstants.ACCOUNT_ENABLED, userName, userStoreDomainName,
+                        null, AuditConstants.AUDIT_SUCCESS,false);
```

Comment:
Shall we keep a space after the comma?

Reply:
Fixed in 0203a22b19ea00df2e1ed6c9ad4f61421349e6eb